src/renderer/components/button/button.scss

```diff
@@ -45,6 +51,7 @@
   &.outlined {
     color: inherit;
     background: transparent;
+    border: 1px solid;
```

Comment:
Please check if apperance of button inside `<InfoPanel/>` stays same because of this change. Search shows that there at least 1 button that using `outlined` style in our codebase.

Reply:
I removed it because it increases the height by 2px.

Reply:
So the old `outlined` styling only worked if the button wasn't also styled the same as the background (namely you couldn't have a `primary outlined` button on `primary` notification. That is why I initially had this change.